### PR TITLE
Feat: Cache Upgrade

### PR DESCRIPTION
**Pull request by Meldiron**
Upgrade cache library from 0.4 to 0.5. This is required if people want to use new 0.5 version of Cache alongside Utopia database

### CODE REVIEW

File: bin/tasks/index.php [hunk 1]
```diff
@@ -26,6 +26,7 @@
     ->param('adapter', '', new Text(0), 'Database adapter', false)
     ->param('name', '', new Text(0), 'Name of created database.', false)
     ->action(function ($adapter, $name) {
+        Cache::setCaseSensitivity(false);
```
Comment (christyjacob4) [inline bin/tasks/index.php:29]: Question for you , if false is meant to be used in most cases, why not use `false` as default ? 🤔  and avoid all these changes?

Comment (Meldiron) [reply bin/tasks/index.php:29]: To prevent breaking change in cache library, mainly.

Comment (christyjacob4) [reply bin/tasks/index.php:29]: How would it be a breaking change in utopia-php/cache if we change this line to false ? 
https://github.com/utopia-php/cache/blob/02ffbe165f9632e05519eb75a2ce32deb19638a0/src/Cache/Cache.php#L15

Comment (Meldiron) [reply bin/tasks/index.php]: New cache version now defaults to false

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**